wifiqr.go

```diff
@@ -1,10 +1,9 @@
 package wifiqr
 
 import (
+	"github.com/skip2/go-qrcode"
 	"strconv"
 	"strings"
-
-	"github.com/skip2/go-qrcode"
```

Comment:
Could you revert the reordering of the imports.

Reply:
done into commit number 5cef7ac